src/hooks/useLogDownloader/index.ts

```diff
@@ -71,12 +72,31 @@ const useLogDownloader = (
         setFileSize(getFileSize() + progress);
       },
     })
-      .then((logs) => {
+      .then(({ result: logs, trimmedLines }) => {
         // Remove the last log line if it is empty
         if (logs[logs.length - 1] === "") {
           logs.pop();
         }
         setData(logs);
+        if (trimmedLines) {
+          sendEvent({
+            downloaded: getFileSize(),
+            duration: Date.now() - timeStart,
+            name: "Log Download Incomplete",
+            reason: "Log line size limit exceeded",
+          });
+          reportError({
+            message: "Log line size limit exceeded",
+            metadata: {
+              trimmedLines,
+              url,
+            },
+            name: "Log download incomplete",
+          }).warning();
+          dispatchToast.warning(LOG_LINE_TOO_LARGE_WARNING, true, {
```

Comment:
Two toasts are firing every time I load the page
<img width="1417" alt="image" src="https://github.com/evergreen-ci/parsley/assets/9298431/793ee41f-d852-4c52-bf62-0ea756d7b366">

Reply:
This is due to React strict mode when running locally. The network request actually fires twice under the hood and thus triggers the toast callback twice. It wouldn't happen on a production build.

Reply:
<img width="1192" alt="image" src="https://github.com/evergreen-ci/parsley/assets/4605522/5b5c4ac2-dd7e-409f-96f1-194f89feaaa2">